GagarinDmitry/src/BuildingBlocks/EventBus/Kafka/Configs/KafkaConfig.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Confluent.Kafka;
```

Comment:
You have same naming in same directory for entity(ConsumerConfig) and for class with extension methods. Please, think of better naming